oak/server/rust/eal/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![cfg_attr(feature="no_std", no_std)]
+
+#[cfg(feature = "asylo")]
+extern crate eal_asylo;
```

Comment:
I thought this is not necessary in Rust 2018 -- or is it still needed for optional deps?

Reply:
You're correct, it's not needed even for optional dependencies.